documentation/reference/color_table.js

```diff
@@ -0,0 +1,116 @@
+    { 'name': "default", 'colors': [ 0x50505080, 0x50505080, 0x50505080, 0x50505080 ] },
+    { 'name': "default", 'colors': [ 0x80540080, 0x80540080, 0x80540080, 0x80540080 ] },
+    { 'name': "default", 'colors': [ 0x70803080, 0x70803080, 0x70803080, 0x70803080 ] },
+    { 'name': "default", 'colors': [ 0x48581080, 0x48581080, 0x48581080, 0x48581080 ] },
+    { 'name': "default", 'colors': [ 0x58605880, 0x58605880, 0x58605880, 0x58605880 ] },
+    { 'name': "default", 'colors': [ 0x40484080, 0x40484080, 0x40484080, 0x40484080 ] },
+    { 'name': "default", 'colors': [ 0x30203080, 0x30203080, 0x30203080, 0x30203080 ] },
+    { 'name': "default", 'colors': [ 0x80794880, 0x80794880, 0x80794880, 0x80794880 ] },
+    { 'name': "default", 'colors': [ 0x205e7880, 0x205e7880, 0x205e7880, 0x205e7880 ] },
+    { 'name': "default", 'colors': [ 0x1d1d1d80, 0x1d1d1d80, 0x1d1d1d80, 0x1d1d1d80 ] },
+    { 'name': "default", 'colors': [ 0x40404080, 0x40404080, 0x40404080, 0x40404080 ] },
+    { 'name': "default", 'colors': [ 0x7a4d6580, 0x7a4d6580, 0x7a4d6580, 0x7a4d6580 ] },
+    { 'name': "default", 'colors': [ 0x7a343480, 0x7a343480, 0x7a343480, 0x7a343480 ] },
+    { 'name': "default", 'colors': [ 0x10654c80, 0x10654c80, 0x10654c80, 0x10654c80 ] },
+    { 'name': "default", 'colors': [ 0x464a7880, 0x464a7880, 0x464a7880, 0x464a7880 ] },
+    { 'name': "default", 'colors': [ 0x577e8080, 0x577e8080, 0x577e8080, 0x577e8080 ] },
+    { 'name': "default", 'colors': [ 0x7f7b3380, 0x7f7b3380, 0x7f7b3380, 0x7f7b3380 ] },
+    { 'name': "default", 'colors': [ 0x79790280, 0x79790280, 0x79790280, 0x79790280 ] },
+    { 'name': "default", 'colors': [ 0x47687a80, 0x47687a80, 0x47687a80, 0x47687a80 ] },
+    { 'name': "default", 'colors': [ 0x40808080, 0x40808080, 0x40808080, 0x40808080 ] },
+    { 'name': "default", 'colors': [ 0x40808080, 0x40808080, 0x40808080, 0x40808080 ] },
+    { 'name': "default", 'colors': [ 0x80808080, 0x80808080, 0x80808080, 0x80808080 ] },
+    { 'name': "default", 'colors': [ 0x20404060, 0x20404060, 0x20404060, 0x20404060 ] },
+    { 'name': "default", 'colors': [ 0x70787080, 0x70787080, 0x70787080, 0x70787080 ] },
+    { 'name': "default", 'colors': [ 0x60000080, 0x60000080, 0x60000080, 0x60000080 ] },
+    { 'name': "default", 'colors': [ 0x80602080, 0x80602080, 0x80602080, 0x80602080 ] },
+    { 'name': "default", 'colors': [ 0x00000180, 0x00000180, 0x00000180, 0x00000180 ] }
+  ]
+}
+
+function jakRgbaToCssRgba(val) {
+  var a = (val >>> 0) & 0xFF,
+      b = (val >>> 8) & 0xFF,
+      g = (val >>> 16) & 0xFF,
+      r = (val >>> 24) & 0xFF;
+  a = a / 128;
+  b = Math.floor(b * 2 * a);
+  g = Math.floor(g * 2 * a);
+  r = Math.floor(r * 2 * a);
+  return "rgba(" + [r, g, b, 1].join(",") + ")";
+}
+
+export default function ColorTable({game}) {
+  var index = -1;
+  return(<div class="language-table container">
+  <tr><th>Index</th><th>Name</th><th>Color</th></tr>
+  { colorTables[game].map( lang => {
```

Comment:
you can get the index in a map via it's second parameter `...map((color, index) => { ...`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map